stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/mention/list/viewmodel/MentionListViewModel.kt

```diff
@@ -39,14 +39,14 @@ public class MentionListViewModel : ViewModel() {
         val isLoading: Boolean,
     )
 
+    private val initialState = State(
```

Comment:
Do we really need this property?
Maybe we can add it to the initialization of `_state`, then, when the state is updated, it is recollected by GC